agr_java_core/src/main/java/org/alliancegenome/neo4j/entity/DiseaseAnnotation.java

```diff
@@ -1,18 +1,34 @@
 package org.alliancegenome.neo4j.entity;
 
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.alliancegenome.api.entity.PresentationEntity;
+import org.alliancegenome.neo4j.entity.node.AffectedGenomicModel;
+import org.alliancegenome.neo4j.entity.node.Allele;
+import org.alliancegenome.neo4j.entity.node.CrossReference;
+import org.alliancegenome.neo4j.entity.node.DOTerm;
+import org.alliancegenome.neo4j.entity.node.ECOTerm;
+import org.alliancegenome.neo4j.entity.node.Gene;
+import org.alliancegenome.neo4j.entity.node.Publication;
```

Comment:
Why are you listing all the classes individually instead of using the 
import org.alliancegenome.neo4j.entity.node.*;
statement? is that an Eclipse thing?
I wished the import statement changes would not all change

Reply:
Eclipse does this... there is an organize imports that runs, it puts them in import order from the jar files.

Reply:
We should harmonize Eclipse and IntelliJ's rules for the imports.

Reply:
Yes I agree... this will save us many headaches